spec/factories/registration.rb

```diff
@@ -3,6 +3,7 @@
 FactoryGirl.define do
   factory :registration do
     user
-    conference
+    conference { create(:conference, registration_period: create(:registration_period, start_date: 3.days.ago)) }
```

Comment:
I don't like this much. I feel that the details regarding the conference and its registration_period need to be explicitely and visibly defined in each test.
@hennevogel  @ChrisBr ?